dev/make/compiler_definitions/gnu.32e.mk

```diff
@@ -1,5 +1,5 @@
 #===============================================================================
-# Copyright 2023 Intel Corporation
+# Copyright 2024 contributors to the oneDAL project
```

Comment:
```suggestion
# Copyright contributors to the oneDAL project
```

Current copyright check expects no year defined and i think this is what was suggested by UXL. 
same true for other files